src/features/metadata-instance-editor/Instance.js

```diff
@@ -617,6 +644,9 @@ class Instance extends React.PureComponent<Props, State> {
         // Animate short and tall cards at consistent speeds.
         const animationDuration = (fields.length + 1) * 50;
 
+        const isExistingAIExtractionCascadePolicy =
+            this.isCascadingEnabled(this.props) && this.isAIFolderExtractionEnabled(this.props);
```

Comment:
can we use different names for these functions, these same names are being used as the booleans for the state or can we have a single source of truth of these flags?